pages.zh/linux/apache2ctl.md

```diff
@@ -0,0 +1,24 @@
+# apache2ctl
```

Comment:
Hrm. Did you mean to translate this page? It looks like English to me.

Reply:
Remove it !